### PR TITLE
localize-datacenter-#2011

### PR DESCRIPTION
**Pull request by matteobisi**
### Describe your changes
I saw an abandoned issue about the translation of this term, so I made a new translation of the term

### Related issue number or link (ex: `resolves #issue-number`)
#2011 

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/it/data-center.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+status: Completed
+category: Tecnologia
+tags: ["infrastruttura", "fondamentali", ""]
+---
+
+## Cos'è
+
+Un data center è un edificio o una struttura specializzata progettata per ospitare più computers, spesso servers.
+I data center sono abitualmente collegati a linee Internet ad alta velocità, soprattutto quando sono focalizzati sul [cloud computing](/it/cloud-computing/). . 
+Gli edifici che ospitano i data center sono attrezzati per mantenere il servizio anche durante eventi avversi, ad esempio con generatori che forniscono energia durante le interruzioni o con potenti sistemi di aria condizionata che mantengono una temperatura controllata, proteggendo i computers dal calore che producono lavorando.
```
Comment (ugho16) [inline content/it/data-center.md:12]:
```suggestion
Gli edifici che ospitano i data center sono attrezzati per mantenere il servizio anche durante eventi avversi, ad esempio con generatori che forniscono energia durante le interruzioni o con potenti sistemi di aria condizionata che mantengono una temperatura controllata, proteggendo i computer dal calore che producono lavorando.
```

**Review by ugho16 — APPROVED** [review]

File: content/it/data-center.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+title: Data center
+status: Completed
+category: Tecnologia
+tags: ["infrastruttura", "fondamentali", ""]
+---
+
+## Cos'è
+
+Un data center è un edificio o una struttura specializzata appositamente progettata per ospitare più computer, spesso server.
+I data center sono abitualmente collegati a linee Internet ad alta velocità, soprattutto quando sono focalizzati sul [cloud computing](/it/cloud-computing/).
```
Comment (sistella) [inline content/it/data-center.md:11]:
```suggestion
I data center sono abitualmente collegati a linee internet ad alta velocità, soprattutto quando sono focalizzati sul [cloud computing](/it/cloud-computing/).
```

File: content/it/data-center.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+Gli edifici che ospitano i data center sono attrezzati per mantenere il servizio anche durante eventi avversi, ad esempio con generatori che forniscono energia durante le interruzioni o con potenti sistemi di aria condizionata che mantengono una temperatura controllata, proteggendo i computer dal calore che producono lavorando.
```
Comment (sistella) [inline content/it/data-center.md:12]:
```suggestion
Gli edifici che ospitano i data center sono attrezzati per mantenere il servizio anche durante eventi avversi, inclusi generatori che forniscono energia durante le interruzioni e potenti sistemi di aria condizionata che mantengono una temperatura controllata, proteggendo i computer dal calore che producono lavorando.
```
più che altro per rimuovere la "o" che sembra quasi esclusiva... che ne dici?

File: content/it/data-center.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+## Quali problematiche affronta
+
+Prima che i data center diventassero prevalenti alla fine degli anni '90, c'erano principalmente computer individuali con compiti specifici o computer utilizzati dalle persone per svolgere il proprio lavoro.
```
Comment (sistella) [inline content/it/data-center.md:16]: A rileggerlo, computer individuali mi dà l'idea di computer dedicati ad una singola persona, mentre qui si vuole indicare le postazioni che erano purpose specifici. Non so, proporrei questo:
```suggestion
Prima che i data center si diffondessero alla fine degli anni '90, i computer si distinguevano principalmente tra quelli dedicati a compiti specifici e quelli utilizzati dalle persone per svolgere il proprio lavoro.
```

**Review by sistella — CHANGES_REQUESTED** [review]

**Review by sistella — APPROVED** [review]

**Pull request merged**